packages/react-avatar/src/components/Avatar/Avatar.scss

```diff
@@ -267,7 +267,12 @@ $default-inactiveScaleFactor: 0.8;
 }
 
 // Activity modifiers
-._active {
+._active,
+._inactive {
+  // Work-around pixel snapping jankiness at the end of the animation if the scale factor is 1.0
+  transform: scale(0.9999);
```

Comment:
Any opinions on this workaround? Without it the avatar jumps at the end of the scale animation.

Reply:
I'd have to pull to see what you are;